src/laser_core/random.py

```diff
@@ -0,0 +1,42 @@
+"""Functions for seeding and accessing the laser-core random number generator."""
+
+import numba as nb
+import numpy as np
+
+_prng = None
+
+
+@nb.jit((nb.uint64,), nopython=True, nogil=True, parallel=True)
+def nbseed(seed):
+    np.random.seed(seed)  # set the non-parallel Numba PRNG seed
```

Comment:
IMO, functions should either be private or have a docstring. (I'm sure it's good for private functions to have doc strings too, but I'd be less dogmatic about that.)

Reply:
Will make private. I'm not sure what happens to Numba-fied functions with docstrings. Maybe a) doc tools just do the right thing and extract the docstring for documentation and b) maybe `nb.jit()` hoists the docstring into the Python part of the Numba-fied function...